### PR TITLE
modify noisebridge.net redirect from index.html to the wiki main page

### PR DESCRIPTION
**Pull request by kevinjos**

### CODE REVIEW

**Comment by SuperQ** [comment]
Nice. It might be worth going and unwrapping the changes that have been made to the `.net` homepage routing over the years. We had a simpler web routing in the past.

**Comment by kevinjos** [comment]
Removed some of the cruft that appears to have been put in place to serve the gala landing page using the mediawiki nginx config example as a reference https://www.nginx.com/resources/wiki/start/topics/recipes/mediawiki/

File: group_vars/noisebridge_net/caddy.yml [hunk 1]
```diff
@@ -48,16 +48,12 @@ caddy_config: |
       X-Content-Type-Options "nosniff"
     }
 
-    fastcgi / unix:/run/php/php7.0-fpm.sock php {
-        except /images
```
Comment (SuperQ) [inline group_vars/noisebridge_net/caddy.yml:52]: Why remove this? This avoids sending static content requests through PHP.

File: group_vars/noisebridge_net/caddy.yml [hunk 2]
```diff
@@ -48,16 +48,12 @@ caddy_config: |
...
-    }
+    fastcgi / unix:/run/php/php7.0-fpm.sock php
 
     rewrite / {
         r /(.*)
-        if {path} not /
-        to {path} {path}/ /index.php?title={1}
+        to /index.php?title={1}
```
Comment (SuperQ) [inline group_vars/noisebridge_net/caddy.yml:55]: This needs to keep the `{path} {path}/` in order for the wiki assets to load correctly.

File: group_vars/noisebridge_net/caddy.yml (hunk 1 above)
Comment (kevinjos) [reply group_vars/noisebridge_net/caddy.yml:52]: Fixed. Thought this was only in place for the old landing page. However, confirmed that media wiki does load images through /images.

File: group_vars/noisebridge_net/caddy.yml (hunk 2 above)
Comment (kevinjos) [reply group_vars/noisebridge_net/caddy.yml]: Fixed.

File: group_vars/noisebridge_net/caddy.yml [hunk 3]
```diff
@@ -49,15 +49,14 @@ caddy_config: |
     }
 
     fastcgi / unix:/run/php/php7.0-fpm.sock php {
-        except /images
+      except /images
     }
 
     rewrite / {
         r /(.*)
         if {path} not /
```
Comment (SuperQ) [inline group_vars/noisebridge_net/caddy.yml:57]: We don't need the `if {path}` line.

Comment (elimisteve) [reply group_vars/noisebridge_net/caddy.yml:57]: Won't we end up with `//` at the beginning of the URL path without it?

Comment (SuperQ) [reply group_vars/noisebridge_net/caddy.yml:57]: No, if you do a little bit of `git blame` spelunking, you can see what this config looked like [before we added the static homepage](https://github.com/noisebridge/infrastructure/blob/8870234f9eb6099f10ef26ae1a00c2399bbaf4a8/group_vars/noisebridge-net/caddy.yml#L57-L65).

**Comment by SuperQ** [comment]
I've manually deployed [the original rewrite config](https://github.com/noisebridge/infrastructure/blob/8870234f9eb6099f10ef26ae1a00c2399bbaf4a8/group_vars/noisebridge-net/caddy.yml#L57-L65) which seems to be working just fine.

If you could please update this, and squash your edits, we can merge this.

**Pull request merged**